dockerfiles/ClipperTestsDockerfile

```diff
@@ -1,4 +1,4 @@
-ARG CODE_VERSION=develop
```

Comment:
I'm guessing that we're removing these default arguments because the images build script, in conjunction with `VERSION.txt` are intended to be the point of control for image versioning?

Reply:
Exactly. And removing the default makes sure that the build fails if we forget a build arg instead of silently building with "develop"